scripts/updateDeployedBytecodes.js

```diff
@@ -1,30 +1,61 @@
 const path = require('path');
 const fs = require('fs');
+const sha256 = require('crypto-js/sha256');

```

Comment:
I think there is a sha256 util in ethers, would be better to use a library we already have